src/cmd/initContainer.go

```diff
@@ -231,7 +231,7 @@ func initContainer(cmd *cobra.Command, args []string) error {
 
 		sudoGroup, err := utils.GetGroupForSudo()
 		if err != nil {
-			return fmt.Errorf("failed to add user %s: %s", initContainerFlags.user, err)
+			return fmt.Errorf("failed to get group for sudo: %w", err)
```

Comment:
I took the liberty to split this typo fix (thanks!) into a [separate commit](https://github.com/containers/toolbox/pull/545)